x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/last_value.tsx

```diff
@@ -287,6 +336,39 @@ export const lastValueOperation: OperationDefinition<LastValueIndexPatternColumn
             }
           />
         </EuiFormRow>
+        <EuiFormRow
+          error={i18n.translate(
+            'xpack.lens.indexPattern.lastValue.showArrayValuesWithTopValuesWarning',
+            {
+              defaultMessage:
+                'When you show array values, you are unable to use this field to rank Top values.',
+            }
+          )}
+          isInvalid={currentColumn.params.showArrayValues && usingTopValues}
+          display="rowCompressed"
+          fullWidth
+          data-test-subj="lns-indexPattern-lastValue-showArrayValues"
+        >
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.lens.indexPattern.lastValue.showArrayValuesExplanation',
+              {
+                defaultMessage:
+                  'Displays all values associated with this field in each last document.',
+              }
+            )}
+            position="left"
+          >
+            <EuiSwitch
```

Comment:
nit: Please use the compressed flag here, we've been doing that for other settings in the sidebar too.